src/templates/layout.hbs

```diff
@@ -34,7 +34,7 @@
         <link rel="icon" type="image/png" sizes="96x96" href="/assets/images/favicon/favicon-96x96.png">
         <link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
 
-        <title>Openscope - Air Traffic Control Simulator</title>
+        <title>openScope Air Traffic Control Simulator</title>
```

Comment:
`Air Traffic Control Simulator` is cased differently here, is that on purpose?

Reply:
I don't quite follow-- different from what?

Reply:
different from lines 10 or 12 where it is all lowercase

Reply:
I see what you're saying:
```
<meta property="Description" content="Free and open source air traffic control simulator"/>
<meta property="og:title" content="openScope Air Traffic Control Simulator"/>
<meta property="og:description" content="Free and open source air traffic control simulator"/>
```

They were cased based on which property they were under.
- I assume description to be somehow used to describe what the site is, so I used normal sentence-style capitalization.
- Title was cased like the title of a book

That was the thought process anwyay.